tools/doxygen/Doxyfile.php

```diff
@@ -817,7 +817,90 @@
 src/php/lib/Grpc/ServerCredentials.php \
 src/php/lib/Grpc/ServerStreamingCall.php \
 src/php/lib/Grpc/Timeval.php \
-src/php/lib/Grpc/UnaryCall.php
+src/php/lib/Grpc/UnaryCall.php \
+src/php/vendor/doctrine/instantiator/CONTRIBUTING.md \
```

Comment:
You probably need to clean up your composer temporary directory before running `generate_projects.sh`.

Reply:
ok, had no experience with doxyfiles...